cmd/convox/env_test.go

```diff
@@ -160,7 +160,7 @@ func TestEnvSetStdin(t *testing.T) {
 		test.Http{
 			Method: "POST",
 			Path:   "/apps/myapp/environment",
-			Body:   "foo=bar\nping=pong\n",
+			Body:   "# Comment followed by blank line\n\nfoo=bar\nping=pong\n",
```

Comment:
I did update the fixture `manifest\fixtures\env-test.env` which I believe is what is fed to the test server - does this mean that this is coded elsewhere?